common/app/model/meta.scala

```diff
@@ -398,6 +400,14 @@ case class GalleryPage(
 
 case class EmbedPage(item: Video, title: String, isExpired: Boolean = false) extends ContentPage
 
+case class MediaAtomEmbedPage(atom: MediaAtom) extends Page {
+  override val metadata = MetaData.make(id = atom.id,
+    webTitle = atom.title,
+    //TODO figure out what the analytics name should be
+    analyticsName = atom.id,
```

Comment:
I think that @mkopka can help you there ! It doesn't seem to be very consistent across the site tho.